packages/rollup/src/utils/versions.ts

```diff
@@ -0,0 +1,5 @@
+export const nxVersion = require('../../package.json').version;
+
+export const swcLoaderVersion = '0.1.15';
+export const swcHelpersVersion = '~0.3.3';
```

Comment:
suggestion: `nrwl/js` also exposes `swcHelpersVersion`. We maybe can use it as we are using `swcCoreVersion` from `nrwl/js` in `init.ts`.